docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,90 @@
+# Scalar DL Backup Creation on Kubernetes
```

Comment:
Backup only? The next sentence says `create and restore Scalar DL backups`, so they are already inconsistent.

Reply:
Also, this doc doesn't even describe how to take a backup, rather it described how to pause and unpause Scalar Ledger pods in k8s. It should describe what it does correctly. Very misleading.
Moreover, it doesn't really describe Scalar DL specific things.

* What should be done to do backup and restore properly in Scalar DL? Any difference when using Sclar DL with only Ledger or with Ledger and Auditor?  In Ledger+Auditor case, pausing Ledger only is fine? If so, why?

Reply:
> Backup only? The next sentence says create and restore Scalar DL backups, so they are already inconsistent.

Yes, Now I feel there is inconsistency (document name also different).

But during the period of document creation and self-review, I thought that this guide mainly handles the backup creation so thought heading `Scalar DL Backup Creation on Kubernetes` is enough.

Reply:
>Also, this doc doesn't even describe how to take a backup, rather it described how to pause and unpause Scalar Ledger pods in k8s. It should describe what it does correctly. Very misleading.
Moreover, it doesn't really describe Scalar DL specific things.
What should be done to do backup and restore properly in Scalar DL? Any difference when using Sclar DL with only Ledger or with Ledger and Auditor? In Ledger+Auditor case, pausing Ledger only is fine? If so, why?

I tried to avoid redundancy. A detailed explanation of backup creation and restoration exists in [A Guide on How to Backup and Restore Data in Scalar DL](https://github.com/scalar-labs/scalardl/blob/master/docs/backup-restore.md) so I specified this document in Prerequisites.

I tried to explain how to create a transactionally consistent backup for Scalar DL on Kubernetes. I thought the continuation of  [A Guide on How to Backup and Restore Data in Scalar DL](https://github.com/scalar-labs/scalardl/blob/master/docs/backup-restore.md)  makes sense here.

>any difference when using Sclar DL with only Ledger or with Ledger and Auditor?

I have added a Requirement `You must identify a unique point-in-time recovery/restore point for the Ledger and Auditor if you use both the Ledger and Auditor.`, and I thought other details will be received from  [A Guide on How to Backup and Restore Data in Scalar DL](https://github.com/scalar-labs/scalardl/blob/master/docs/backup-restore.md) 

>In Ledger+Auditor case, pausing Ledger only is fine? 

In my understanding, yes